services/blockchain-indexer/shared/dataService/business/tokens.js

```diff
@@ -133,46 +124,31 @@ const getTopLiskAddresses = async (params) => {
 	return topLiskAddresses;
 };
 
-const getSupportedTokens = async (params) => {
-	const supportedTokensList = {
-		data: {
-			supportedTokens: [],
-		},
+const getTokensSummary = async () => {
+	const summary = {
+		data: {},
 		meta: {},
 	};
 
-	const response = await requestConnector('token_getSupportedTokens');
+	const escrowedAmount = await requestConnector('token_getEscrowedAmounts');
+	const supportedTokens = await requestConnector('token_getSupportedTokens');
+	const totalSupply = await requestConnector('token_getTotalSupply');
```

Comment:
The response of the these endpoints is object which passed as an input to `populateTokenMetaInfo` which require array.
```suggestion
	const { escrowedAmounts } = await requestConnector('token_getEscrowedAmounts');
	const { tokenIDs: supportedTokens } = await requestConnector('token_getSupportedTokens');
	const { totalSupply } = await requestConnector('token_getTotalSupply');
```

Reply:
Thanks for the catch. It is resolved.